### PR TITLE
Minor UI Changes - Experimental Alpha

### PR DESCRIPTION
**Pull request by daxit**
-Updated images using material.io (Google Open Source Icons)
-Updated some fonts
-Updated spacing, and colors

Uses the 1.5.2 experimental build. Let me know if I broke anything :)

### CODE REVIEW

**Comment by daxit** [comment]
Thank you for letting me know. I have added this back in. Please let me know if there is anything else.

File: Libs/PandoraSharp/Song.cs [hunk 1]
```diff
@@ -49,7 +49,6 @@ public Song(Pandora p, JToken song)
 
             AmazonAlbumID = (string)song["amazonAlbumDigitalAsin"];
             AmazonTrackID = (string)song["amazonSongDigitalAsin"];
-            AmazonAlbumUrl = (string)song["amazonAlbumUrl"];
```
Comment (Wires77) [inline Libs/PandoraSharp/Song.cs]: These changes in this file will have to be reverted, as they fixed an issue in another pull request.

**Comment by Wires77** [comment]
I'll take a look at this when I get a chance on my local machine.  The one thing I want to check is if the new icons match those in the notify menu (Found by right-clicking the system tray icon), the JumpList menu (Found by right-clicking the taskbar icon once pull request #167 is merged), and the media key icons (Found by hovering over the taskbar icon when Elpis is running).  Otherwise, judging from the screenshot in #123 this looks pretty good.

**Comment by Wires77** [comment]
Looks like there are a few pull requests that these changes would end up reverting.  If you can apply those to your branch or revert the changes you pushed for the .cs files, this should be good.

**Comment by daxit** [comment]
Thank you for looking this over.  I will update this with reverted cs files in a few days.

Also I did notice right clicking the tray icon still has the black info icon, but I am not sure about how to change this.

**Comment by adammhaile** [comment]
@Wires77 Email me.... adammhaile@gmail.com
I realized that you don't really have any of the details for how to do a new release of this and there's a special file you need (that's not in the git repo) for building a release binary.

**Comment by daxit** [comment]
I replaced the cs files with the latest from the Elpis Master branch. Hopefully this fixes the relevant issues. Let me know if there is anything else.

**Pull request merged**